test/items/temp-stat-stage-booster.test.ts

```diff
@@ -138,7 +138,7 @@ describe("Items - Temporary Stat Stage Boosters", () => {
       "SelectModifierPhase",
       UiMode.MODIFIER_SELECT,
       () => {
-        const handler = game.scene.ui.getHandler() as ModifierSelectUiHandler;
+        const handler = game.scene.ui.getCurrentHandler() as ModifierSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        const handler = game.scene.ui.getCurrentHandler<ModifierSelectUiHandler>();
```